docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -248,6 +248,8 @@ SHOW PLACEMENT LIKE 'POLICY standardpol%';
 
 This will match for `PLACEMENT POLICY` names such as `standardpolicy`.
 
+The `scheduling_state` is one of `SCHEDULED`, `INPROGRESS`, or `PENDING`. `PENDING` means the placement rule is valid, but might not be able to be scheduled based on the current topology of the cluster.
```

Comment:
It does not mean it is strictly valid. `PENDING` merely means it is not schedulable now. The previous `isolationLevel` rules are invalid, but will pass the validation checks of PD, then it goes into `PENDING` forever whatever the topology is.

BTW, does `show placement` needs an example of `PENDING`?

Reply:
I will change it from "valid" to "semantically valid". This is also accurate, since we do some validation before sending requests to PD in the first place.

I don't think we need an example, since the design doc is quite long already.